core/src/main/java/org/elasticsearch/action/bulk/BulkItemRequest.java

```diff
@@ -94,15 +94,36 @@ public void readFrom(StreamInput in) throws IOException {
         request = DocWriteRequest.readDocumentRequest(in);
         if (in.readBoolean()) {
             primaryResponse = BulkItemResponse.readBulkItem(in);
+            if (in.getVersion().onOrAfter(Version.V_5_6_0_UNRELEASED)) {
```

Comment:
Agreed - I wanted to add a 6.0.0 here and keep the wire level logic symmetrical. The 6.0.0 constant is not available on 5.x, which I wanted to discuss / understand why.